recOrder/plugin/main_widget.py

```diff
@@ -234,6 +234,7 @@ def __init__(self, napari_viewer: Viewer):
         self.ui.cb_phase_denoiser.currentIndexChanged[int].connect(
             self.enter_phase_denoiser
         )
+        self.enter_phase_denoiser()
```

Comment:
This will default `MainWidget.phase_regularizer` to `"Tikhonov"`. I think this behavior is fine, yet not sure if we should declare it explicitly.